servicetalk-client-api/src/main/java/io/servicetalk/client/api/AbstractRetryingFilterBuilder.java

```diff
@@ -48,6 +49,7 @@
 public abstract class AbstractRetryingFilterBuilder<Builder
         extends AbstractRetryingFilterBuilder<Builder, Filter, Meta>, Filter, Meta> {
     private static final Duration FULL_JITTER = ofDays(1024);
+    private static final Duration NULL_JITTER = ofNanos(1);
```

Comment:
Perhaps MIN_JITTER as it is still some (albeit trifling) jitter. Would a larger value, ofMillis(1) perhaps, be a better default for the cases where we are providing jitter value?